src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import duke.exception.TaskNotFoundException;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskListTest {
+    private final TaskList tasks = new TaskList();
+
+    @Test
+    public void testAddTask() {
+        assertEquals(0, tasks.getNumTasks());
+
+        tasks.addTask(new Todo("new task"));
+        assertEquals(1, tasks.getNumTasks());
+    }
+
+    @Test
+    public void testGetTask() {
```

Comment:
Maybe you can have a look at the naming convention mentioned in week 3 notes 3.7:

> Java code normally use camelCase for method names e.g., `testStringConversion` but when writing test methods, sometimes another convention is used:
> `unitBeingTested_descriptionOfTestInputs_expectedOutcome`
> e.g., `intDivision_zeroDivisor_exceptionThrown`

but I believe your naming looks good as well!